lib/tasks/manageiq/ui_tasks.rake

```diff
@@ -39,11 +39,23 @@ if Rake::Task.task_defined?("assets:precompile")
   Rake::Task["assets:precompile"].enhance do
     Rake::Task["webpack:compile"].invoke
   end
+
+  Rake::Task["assets:precompile"].actions.each do |action|
+    if action.source_location[0].include?(File.join("lib", "tasks", "webpacker"))
+      Rake::Task["assets:precompile"].actions.delete action
```

Comment:
I feel a bit dirty removing someone else's action... In what circumstance does their action make sense?